.github/workflows/pr.yml

```diff
@@ -20,4 +20,4 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - run: npm run build --if-present
-      - run: npm test
+      - run: TEST_ACCOUNT_PRIVATE_KEY=${{ secrets.TEST_ACCOUNT_PRIVATE_KEY }} npm test
```

Comment:
lets use the env key on the job instead inline?
https://docs.github.com/en/actions/learn-github-actions/environment-variables#using-contexts-to-access-environment-variable-values